installation/scripts/install-db.sh

```diff
@@ -51,7 +51,7 @@ do
 done
 set -- "${POSITIONAL[@]}" # restore positional parameters
 
-echo "Install DB"
+echo "Installing DB"
 helm upgrade --install --atomic --debug --timeout "${TIMEOUT}" -f ./mergedOverrides.yaml --create-namespace --namespace compass-system localdb "${DB_CHARTS}"
```

Comment:
As you are already removing the spam from the logs, maybe remove the `--debug` flag to exclude verbose output

Reply:
not sure what was the initial motivation to use this flag but when I compared the logs most of the debug logs are about the helm charts/values.yaml. there are couple with resource creation info. Nevertheless, I remove the debug flag.